tests/system/orchestrator-ci-system.conf.json

```diff
@@ -76,6 +76,9 @@
   "PostGracefulTakeoverProcesses": [
     "echo 'Planned takeover complete' >> /tmp/recovery.log"
   ],
+  "PostUnsuccessfulGracefulTakeoverProcesses": [
```

Comment:
Suggestion: maybe it would be better to keep everything inside the test directory and apply this config parameter only for the new test?

Reply:
Thank you, moved.